### PR TITLE
successfully introduced the complete accountant model.

### LINKED ISSUE #1: issue/1
Creating Register,login and Transaction module.

### PR DESCRIPTION
**Pull request by vijay-singh07**
Resolves - #1 

### Description:
- The Register and Login module has been Created.
- The Transaction and Transaction Listing module has been created that is only accessible after a user is logged in.
- Middleware is successfully added to the whole module.
- The jsPDF is also successfully installed and working with Transaction module.

### Screenshots:-

**Register Page:-**
image

**Login Page:-**
image

**Transaction Listing Page**
image

**Create Transaction Page:-**
image

**Edit Transaction Page**
image

### CODE REVIEW

File: app/Http/Controllers/TransactionController.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Inertia\Inertia;
+use Illuminate\Http\Request;
+use App\Models\transaction;
```
Comment (Gauravbhatt19) [inline app/Http/Controllers/TransactionController.php]:
```suggestion
use App\Models\Transaction;
```
Also rename `app\Models\transaction.php` file as `app\Models\Transaction.php`

File: app/Http/Controllers/TransactionController.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            $Transaction->paid = $request->paid;
+            $Transaction->unit_amount = $request->unit_amount;
+            $Transaction->unit_quantity = $request->unit_quantity ;
+            $Transaction->unit_name = $request->unit_name;
+            $Transaction->type = $request->type;
+            $Transaction->status = $request->status;
+            $Transaction->utr = $request->utr;
+            $Transaction->comments = $request->comments;
+            $Transaction->project = $request->project;
+            $res= $Transaction-> save();
```
Comment (Gauravbhatt19) [inline app/Http/Controllers/TransactionController.php]:
```suggestion
            $res = $Transaction->save();
```

File: app/Http/Controllers/TransactionController.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    public function index(){
+        return Inertia::render('CreateTransaction');
+    }
+
+    public function show(){
+        return Inertia::render('TransactionList');
+    }
+
+    public function save(TransactionRequest $request){
+            $Transaction= new Transaction();
```
Comment (Gauravbhatt19) [inline app/Http/Controllers/TransactionController.php]:
```suggestion
            $Transaction = new Transaction();
```

File: app/Http/Controllers/TransactionController.php [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            $Transaction->description = $request->description;
+            $Transaction->paid = $request->paid;
+            $Transaction->unit_amount = $request->unit_amount;
+            $Transaction->unit_quantity = $request->unit_quantity ;
+            $Transaction->unit_name = $request->unit_name;
+            $Transaction->type = $request->type;
+            $Transaction->status = $request->status;
+            $Transaction->comments = $request->comments;
+            $Transaction->project = $request->project;
+            $res= $Transaction-> save();
```
Comment (Gauravbhatt19) [inline app/Http/Controllers/TransactionController.php]:
```suggestion
            $res = $Transaction->save();
```

File: app/Http/Controllers/TransactionController.php [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    public function delete($id){
+        $editable = Transaction::find($id);
+            $editable->delete();
+        return Redirect::route('transaction-list');
+    }
+
+    public function update( Request $request){
+            $Transaction= Transaction::find($request->id);
```
Comment (Gauravbhatt19) [inline app/Http/Controllers/TransactionController.php]:
```suggestion
            $Transaction = Transaction::find($request->id);
```

File: app/Http/Controllers/TransactionController.php [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+            $Transaction-> date= $request->date;
```
Comment (Gauravbhatt19) [inline app/Http/Controllers/TransactionController.php]:
```suggestion
            $Transaction->date = $request->date;
```

File: app/Models/transaction.php [hunk 7]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class transaction extends Model
```
Comment (Gauravbhatt19) [inline app/Models/transaction.php]:
```suggestion
class Transaction extends Model
```

**Review by Gauravbhatt19 — APPROVED** [review]
Minor formatting & casing suggestions, rest looks good to merge 👍🏻

File: app/Http/Controllers/TransactionController.php (hunk 2 above)
Comment (vijay-singh07) [reply app/Http/Controllers/TransactionController.php]: Thank you for the suggestions. The changes shall reflect by latest commit.

**Review by AmanKala — APPROVED** [review]

**Pull request merged**